File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_2/exercice10.c
#include <stdio.h>
float N1,N2,solution;
char op;
int main(){
    printf("saisir deux nombres N1 et N2 : \n");
    scanf("%f %f",&N1,&N2);
    printf("saisir une opération ( +, - , / , * ): ");
    scanf(" %c",&op);
    if(op == '+'){
        solution=N1+N2;
        printf("%f + %f = %f",N1,N2,solution);
    }
    else if(op == '-'){
        solution=N1-N2;
        printf("%f - %f = %f",N1,N2,solution);
    }
    else if(op == '*'){
        solution=N1*N2;
        printf("%f * %f = %f",N1,N2,solution);
    }
    else if(op == '/'){
        if (N2!=0){
            solution=N1/N2;
            printf("%f / %f = %f",N1,N2,solution);
        }
        else
            printf("math error!\n");
    }
    else
        printf("invalide choix.\n");
    return 0;
}

File: LES_SERIE_S1_IA_FS_ALGO_PYTHON_C/Série_de_TD-TP_2/exercice10.py
N1=float(input("saisir le nombre 1: "))
N2=float(input("saisir le nombre 2: "))
op=str(input("saisir une opération ( +, - , / , * ): "))[0]
if(op == '+'):
    solution=N1+N2
    print(f"{N1} + {N2} = {solution}")
elif(op == '-'):
    solution=N1-N2
    print(f"{N1} - {N2} = {solution}")
elif(op == '*'):
    solution=N1*N2
    print(f"{N1} * {N2} = {solution}")
elif(op == '/'):
    if(N2!=0):
        solution=N1+N2
        print(f"{N1} + {N2} = {solution}")
    else:
        print("math error!")
else:
    print("invalide choix.")
